tests/src/core/testqgssnappingutils.cpp

```diff
@@ -108,10 +131,16 @@ class TestQgsSnappingUtils : public QObject
       snappingConfig.setType( QgsSnappingConfig::Vertex );
       u.setConfig( snappingConfig );
 
-      QgsPointLocator::Match m = u.snapToMap( QPoint( 100, 100 ) );
+      QgsPointLocator::Match m = u.snapToMap( QPoint( 11, 11 ) );
+      QVERIFY( !m.isValid() );
+      QVERIFY( !m.hasVertex() );
+
+      u.setEnableSnappingForInvisibleFeature( true );
+      mVL->styleChanged();
```

Comment:
It should not be necessary to emit styleChanged() here - the setEnableSnappingForInvisibleFeature() should take care of the invalidation of point locators.

Reply:
yes right,